### DISCUSSION TITLE
How to load object masks from label file as tensors?

### DISCUSSION THREAD

**Discussion by 7f0c** [ultralytics/ultralytics#8136] [Q&A]
I have a segmentation dataset following the dataset format indicated [here](https://docs.ultralytics.com/datasets/segment/#supported-dataset-formats). What I want to do is load these masks as tensors. How do I do that?

**Comment by glenn-jocher** [comment]
@7f0c segmentation datasets must be provided in the polygon format specified in https://docs.ultralytics.com/datasets/segment/#supported-dataset-formats, it is not possible to provide masks directly as this would be prohibitively large for very large segmentation datasets.
